test/common.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const fs = require('fs');
+
+const REPORT_SECTIONS = [
+  'NodeReport',
+  'JavaScript Stack Trace',
+  'JavaScript Heap',
+  'System Information'
+];
+
+exports.locate = (pid) => {
```

Comment:
its helpful to have a comment showing the file format you are looking for with this kind of code